iOSClient/Media/NCMediaCommandView.swift

```diff
@@ -28,7 +28,8 @@ class NCMediaCommandView: UIView {
 
     @IBOutlet weak var title: UILabel!
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
-    @IBOutlet weak var selectButton: UIButton!
+    @IBOutlet weak var selectcancelButton: UIButton!
```

Comment:
Should be selectCancel

Reply:
sure, but they are still separate words, so camel case is needed. maybe `selectOrCancel` would work